### DISCUSSION TITLE
Unexpected git clone command on terminal

### DISCUSSION THREAD

**Discussion by visjble** [community/community#50282] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

I was in the process of building a simple ChatGPT bot for my Discord server using Python commands on Ubuntu. While the bot was running, I noticed an unexpected line of code that randomly appeared. It was a Git clone command that I did not initiate, which attempted to clone a repository. Unfortunately, I was unable to copy the repository link, but I remember that it contained the words 'mariuszkar...trello... .git'. I immediately searched my directories for any newly cloned packages but did not find any. I also tried to retrieve that clone command through a reverse search in the terminal, but nothing appeared.

Could you please advise me on what to do in this situation and let me know if there are specific Git files that I should look for on my system that may be undesirable?

Thank you for your help. I am concerned about the possibility of having an infected machine.

**Comment by Anmol-Baranwal** [comment]
It sounds like you may have experienced a security breach on your machine or in your Discord server. The unexpected Git clone command could have been initiated by an attacker who gained access to your system or Discord account.

Check for any unauthorized changes: Review all of your files and directories to check for any new or modified files that you did not create. This can help identify any malicious files that may have been installed on your system.

Contact Discord support: If you suspect that your Discord account has been compromised, contact Discord support immediately to report the incident and request assistance.
> This is just my guess. I am not sure if this is really the case.

**Reply by Anmol-Baranwal** [reply]
@visjble 
I just wanted to check in and see if you had any further questions or concerns about my answer on GitHub. If you have no further doubts or if your doubts have been resolved, would you mind marking my answer as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by visjble** [comment]
I was actually looking for more specific troubleshooting directives. I did not see any suspicious files using `find . -type f -mtime -2`, which lists modified files in those last 2 days. Also,  running `git reflog --since=7.days` did not show anything suspicious in the git commands executed in the last 7 days, with the most recent commands at the top. I deleted the discord app, changed the discord password. Hope this helps someone else.
